NitroxClient/MonoBehaviours/NitroxEntity.cs

```diff
@@ -61,6 +94,27 @@ public static void SetNewId(GameObject gameObject, NitroxId id)
                 entity = gameObject.AddComponent<NitroxEntity>();
             }
 
+            List<GameObject> childGameObjects = new List<GameObject>();
+
+            if (gameObjectsWaitingForParent.TryGetValue(id, out childGameObjects))
+            {
+                foreach (GameObject childGameObject in childGameObjects)
+                {
+                    if (childGameObject == null)
```

Comment:
How will this code interact with the simulation framework?  Will we still keep the request to simulate if an entity is pending?